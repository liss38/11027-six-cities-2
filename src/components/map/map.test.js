import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map.jsx';

it(`Map component is rendered`, () => {
  const div = global.document.createElement(`div`);

  div.setAttribute(`id`, `map`);
  global.document.body.appendChild(div);

  const tree = renderer
    .create(<Map
      locations={[]}
      styleMode={`main`}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
