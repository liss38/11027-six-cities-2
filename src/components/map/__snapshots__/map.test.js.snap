// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map component is rendered 1`] = `
<section
  className="cities__map map"
  id="map"
/>
`;
